js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/GraphNetwork.tsx

```diff
@@ -109,16 +111,18 @@ export const GraphNetwork = () => {
       background="white"
       position="relative"
     >
-      <GraphRightClickMenu
-        portalProps={{
-          containerRef
-        }}
-      />
-      <Box id="graphNetworkWrap" height="full" backgroundColor="white">
-        {progressValue < 100 ? <Progress value={progressValue} /> : null}
-        <Box ref={visJsRef} height="calc(100% - 40px)" width="full" />
-        <GraphInfo />
-      </Box>
+      {!isSmart? <>
+        <GraphRightClickMenu
```

Comment:
let's avoid this nesting. we can move this into it's own `<GraphNetworkInner />` component and then do `{!isSmart ? <GraphNetworkInner /> : <SmartGraphEmptyState /> }` like in other places

Reply:
Refactored it into a separate component.